crates/nu-path/Cargo.toml

```diff
@@ -13,9 +13,12 @@ bench = false
 
 [dependencies]
 dirs = { workspace = true }
+cfg-if = "1.0.0"
 
 [target.'cfg(windows)'.dependencies]
 omnipath = { workspace = true }
+once_cell = "1.20.1"
```

Comment:
It would be great not to use once_cell since std already has a similar built-in function.

Reply:
OK